### PR TITLE
Allows blurb to correctly split paths on Windows

### PR DESCRIPTION
**Pull request by zooba**

### CODE REVIEW

File: blurb/blurb.py [hunk 1]
```diff
@@ -519,7 +519,7 @@ def _parse_next_filename(filename):
 Parses a "next" filename into its equivalent blurb metadata.
 Returns a dict.
         """
-        components = filename.split("/")
+        components = filename.split(os.sep)
```
Comment (zware) [inline blurb/blurb.py]: Should we also `os.path.normpath`?

Comment (zooba) [reply blurb/blurb.py]: They're all obtained via glob, so it isn't going to be necessary.

**Comment by zooba** [comment]
@larryhastings This is the change - it's been merged, so just need the `setup.py sdist upload` step

**Comment by larryhastings** [comment]
Well, one more step needed: I had to increment the version number.  blurb 1.0.5 is now pushed to the cheeseshop.

**Pull request merged**